### PR TITLE
[vspd] Add processing of Managed and Unmanaged tickets

### PR DESCRIPTION
**Pull request by vctt94**
~Requires on #3120~

This diff adds processing of managed tickets and unmanaged (solo) tickets.  Now on restores 
first process managed it called and updates the status of any live ticket that is found across
all available VSPs.  Then if any are still found but not managed, they are given the chance
to process unmanaged and get their fees paid for solo tickets.

Also checks are done on normal starts/restarts to make sure all live tickets are properly managed
and without error.

### CODE REVIEW

File: app/actions/VSPActions.js [hunk 1]
```diff
@@ -37,6 +38,7 @@ export const getVSPTicketsByFeeStatus = (feeStatus) => (dispatch, getState) => {
   dispatch({ type: GETVSPTICKETSTATUS_ATTEMPT });
   wallet.getVSPTicketsByFeeStatus(getState().grpc.walletService, feeStatus)
     .then(response => {
+      console.log(response);
```
Comment (alexlyp) [inline app/actions/VSPActions.js:41]: remove log

File: app/actions/VSPActions.js [hunk 2]
```diff
@@ -505,3 +522,85 @@ export const setRememberedVspHost = (rememberedVspHost) => (dispatch, getState)
...
+    if (mixedAccount) {
+      feeAccount = mixedAccount;
+      changeAccount = sel.getChangeAccount(getState());
+    } else {
+      feeAccount = sel.defaultSpendingAccount(getState());
+      changeAccount = sel.defaultSpendingAccount(getState());
+    }
+
+    console.log(feeAccount)
+    console.log(changeAccount)
```
Comment (alexlyp) [inline app/actions/VSPActions.js:593]: remove log

File: app/actions/VSPActions.js [hunk 3]
```diff
@@ -505,3 +522,85 @@ export const setRememberedVspHost = (rememberedVspHost) => (dispatch, getState)
...
+    const mixedAccount = sel.getMixedAccount(getState());
+    if (mixedAccount) {
+      feeAccount = mixedAccount;
+      changeAccount = sel.getChangeAccount(getState());
+    } else {
+      feeAccount = sel.defaultSpendingAccount(getState());
+      changeAccount = sel.defaultSpendingAccount(getState());
+    }
+
+    console.log(feeAccount)
```
Comment (alexlyp) [inline app/actions/VSPActions.js:592]: remove log

File: app/stateMachines/GetStartedStateMachine.js [hunk 4]
```diff
@@ -311,12 +311,36 @@ export const getStartedMachine = Machine({
         }
       }
     },
+    // XXX
```
Comment (alexlyp) [inline app/stateMachines/GetStartedStateMachine.js:314]: comment still apply?

File: app/wallet/control.js [hunk 5]
```diff
@@ -332,3 +332,69 @@ export const getPeerInfo = (walletService) => new Promise((ok, fail) => {
...
+        reject(error);
+      }
+
+      const request = new api.ProcessManagedTicketsRequest();
+      request.setVspHost("https://" + vspHost);
+      request.setVspPubkey(vspPubkey);
+      request.setFeeAccount(feeAccount);
+      request.setChangeAccount(changeAccount);
+
+      console.log(walletService);
```
Comment (alexlyp) [inline app/wallet/control.js:352]: remove log

File: app/wallet/control.js [hunk 6]
```diff
@@ -332,3 +332,69 @@ export const getPeerInfo = (walletService) => new Promise((ok, fail) => {
     err ? fail(err) : ok({ ...res })
   );
 });
+
+export const startVSPClient = (walletService, passphrase, vspHost, vspPubkey, feeAccount, changeAccount) =>
```
Comment (alexlyp) [inline app/wallet/control.js:336]: probably should rename to ProcessManagedTickets

File: app/wallet/control.js [hunk 7]
```diff
@@ -332,3 +332,69 @@ export const getPeerInfo = (walletService) => new Promise((ok, fail) => {
...
+      reject(error);
+    }
+
+    const request = new api.ProcessUnmanagedTicketsRequest();
+    request.setVspHost("https://" + vspHost);
+    request.setVspPubkey(vspPubkey);
+    request.setFeeAccount(feeAccount);
+    request.setChangeAccount(changeAccount);
+
+    console.log(walletService);
```
Comment (alexlyp) [inline app/wallet/control.js:385]: remove log

**Review by alexlyp — CHANGES_REQUESTED** [review]

File: app/stateMachines/GetStartedStateMachine.js (hunk 4 above)
Comment (vctt94) [reply app/stateMachines/GetStartedStateMachine.js:314]: Yeah, it does. 

I will come back to it later after release

Comment (alexlyp) [reply app/stateMachines/GetStartedStateMachine.js:314]: kk

**Review by alexlyp — APPROVED** [review]

**Pull request merged**